### DISCUSSION TITLE
PNP Search Results - PDF Preview Cancel icon

### DISCUSSION THREAD

**Discussion by JACKIEOSL** [microsoft-search/pnp-modern-search#2677] [Q&A]
PNP Search Results - PDF Preview Cancel icon
I am using a simple list result. The preview image that opens has 2 cancel icons when opening a PDF. When selecting the icon in the image it returns the user to the library and not back to the results page. Anyone know how to change that?
![thumbnailpreviewPDF](https://user-images.githubusercontent.com/62598973/210602266-15cf5380-1ec8-4176-95fe-36b46ce1b14a.PNG)

**Comment by wobba** [comment]
The inner X is provided by the SharePoint UX. You can try to modify the template and craft a different URL to be used for the preview - see if you can add some parameter to hide the top navigation in the SharePoint preview component.

**Reply by JACKIEOSL** [reply]
I am not versed in modifying the template code that would affect this. I have looked at it but cannot identify where I would apply new code.
I was hoping someone in the community had an actual sample code and instruction on where to add it.

**Comment by JACKIEOSL** [comment]
We still have not resolved this issue. Hoping to hear from anyone on how to accomplish the desired outcome with specifics.
